wis2box_api/plugins/process/bufr2geojson.py

```diff
@@ -106,9 +107,13 @@ def execute(self, data):
         try:
             if 'data_url' in data:
                 data_url = data['data_url']
+                # replace the public URL with the internal minio URL
+                LOGGER.debug('Replacing STORAGE_PUBLIC_URL with internal minio URL')  # noqa
+                # data is stored in the wis2box-public bucket at http://minio:9000/wis2box-public  # noqa
+                data_url = data_url.replace(STORAGE_PUBLIC_URL, 'http://minio:9000/wis2box-public')  # noqa
```

Comment:
Could the internal storage URL (i.e. the internal Docker hostname) be an environment variable (not critical/we can address later)?